_raml/apps/products/examples/patch_product_productid_response.json

```diff
@@ -83,6 +83,119 @@
     "manufacturerPrice": null,
     "basePrice": null
   },
+  "bulkPriceInfo": [
+    {
+      "quantity": {
+        "amount": 2,
+        "unit": "Stück"
+      },
+      "price": {
+        "taxType": "GROSS",
+        "formatted": "155,00 €",
```

Comment:
@mhoehn The same thing as in _raml/apps/products/examples/get_product_updated_property_response.json applies here.